modules/eks/cluster/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## Release 1.446.0
+
+Components PR [#1046](https://github.com/cloudposse/terraform-aws-components/pull/1046)
+
+Added support for passing extra arguments to `kubelet` and other startup modifications supported by EKS on Amazon Linux
+2 via the
+[`bootsrap.sh`](https://github.com/awslabs/amazon-eks-ami/blob/d87c6c49638216907cbd6630b6cadfd4825aed20/templates/al2/runtime/bootstrap.sh)
+script.
```

Comment:
Update the README with example usage of passing extra arguments

Reply:
Done